utils.py

```diff
@@ -503,6 +504,37 @@ def convert_to_str(string_to_convert):
         return string_to_convert.encode('utf-8')
     return string_to_convert
 
+def get_sorted_nodes(nodes):
```

Comment:
Perhaps we should define this on a collection object: collection.get_sorted_nodes(). This file is definitely the wrong place for this function, since the function pertains mainly to the collection object. Only use utils.py for general-purpose functions that are broadly applicable and do not relate specifically to particular constructs in Oppia.